sympy/solvers/tests/test_solveset.py

```diff
@@ -823,11 +819,11 @@ def test_solve_trig():
                   9*sqrt(57))**Rational(1, 3))/(3*(67 + 9*sqrt(57))**Rational(1, 6))) +
                   2*pi), S.Integers))
 
-    assert solveset_real(2*tan(x)*sin(x) + 1, x) == Union(
-        ImageSet(Lambda(n, 2*n*pi + atan(sqrt(2)*sqrt(-1 +sqrt(17))/
-            (1 - sqrt(17))) + pi), S.Integers),
-        ImageSet(Lambda(n, 2*n*pi - atan(sqrt(2)*sqrt(-1 + sqrt(17))/
-            (1 - sqrt(17))) + pi), S.Integers))
+    # assert solveset_real(2*tan(x)*sin(x) + 1, x) == Union(
+    #     ImageSet(Lambda(n, 2*n*pi + atan(sqrt(2)*sqrt(-1 +sqrt(17))/
```

Comment:
What happened with this tests?

Reply:
I don't know, I still have to go through them in more detail. For now I've just re-used those of the older PRs in order to give attribution and to have a starting point. (It also took care of the changed doctests.)